src/qml/filters/hue/ui_frei0r.qml

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2014-2015 Meltytech, LLC
+ * Author: Brian Matherly <pez4brian@yahoo.com>
```

Comment:
I am not the author :)

Reply:
Haha, sorry, I am new to this. Actually what should I put in Copyright? and do I have to put an email?

Reply:
Copyright (c) 2018 Meltytech, LLC

And you can leave out the authorship info. We should start doing that, in general, Brian as we have git log and typically multiple authors.

Reply:
I was thinking the same thing. Copyright is important, author is not.